vcsclient/vcsclient.go

```diff
@@ -90,6 +90,11 @@ type VcsClient interface {
 	// repository   - VCS repository name
 	// branch       - The name of the branch
 	GetLatestCommit(ctx context.Context, owner, repository, branch string) (CommitInfo, error)
+
+	// GetCloneInfo Returns a information that can be used to clone the repository.
+	// owner        - User or organization
+	// repository   - VCS repository name
+	GetCloneInfo(ctx context.Context, owner, repository string) (CloneInfo, error)
```

Comment:
Should this be something like `GetRepositoryInfo` and for the moment it only contains clone URLs? This would allow us to enrich the response later if we need more details.